xml_converter/src/string_helper.hpp

```diff
@@ -14,4 +14,12 @@ vector<string> split(string input, string delimiter);
 
 string normalize_type_name(string type_name);
 
+#ifndef _BASE64_H_
```

Comment:
No need for the include guards here. You probably copied these from a full header file which included them as boilerplate.